nixos/modules/security/pam.nix

```
@@ -886,6 +904,7 @@ in
         vlock = {};
         xlock = {};
         xscreensaver = {};
+        fscrypt = {};
```

```suggestion
        fscrypt.text = "auth        required    pam_unix.so";
```

Maybe this would be better (https://github.com/google/fscrypt#allowing-fscrypt-to-check-your-login-passphrase)?